### PR TITLE
Treatment Changes - Darwin Release

### PR DESCRIPTION
**Pull request by wwysoc2**

### CODE REVIEW

File: gdcdictionary/schemas/treatment.yaml [hunk 1]
```diff
@@ -130,8 +199,14 @@ properties:
...
+      - Intrahepatic
+      - Intramuscular
+      - Intraperitoneal
       - Intrathecal
+      - Intratumor
       - Intravenous
+      - Intravesical
+      - Subcutaneous
       - Oral
       - Not Reported
```
Comment (wteouchicago) [inline gdcdictionary/schemas/treatment.yaml:212]: just noting we didn't do array of enums for this prop.

File: gdcdictionary/schemas/treatment.yaml [hunk 2]
```diff
@@ -68,6 +69,29 @@ properties:
         $ref:
           - "_terms_enum.yaml#/not_reported/common"
 
+  clinical_trial_indicator:
+    $ref:
+      - "_terms.yaml#/clinical_trial_indicator/common"
+    enum:
+      - "Yes"
+      - "No"
+      - Not Reported
```
Comment (wteouchicago) [inline gdcdictionary/schemas/treatment.yaml:78]: jira ticket says this should be Unknown?

File: gdcdictionary/schemas/treatment.yaml [hunk 3]
```diff
@@ -130,8 +199,14 @@ properties:
     $ref:
       - "_terms.yaml#/route_of_administration/common"
     enum:
+      - Intrahepatic
+      - Intramuscular
+      - Intraperitoneal
       - Intrathecal
+      - Intratumor
```
Comment (wwysoc2) [inline gdcdictionary/schemas/treatment.yaml:206]: @tlicht3 , should this be "Intratumoral"?  Google seems to think so, but Google is often wrong.

File: gdcdictionary/schemas/treatment.yaml (hunk 2 above)
Comment (wwysoc2) [reply gdcdictionary/schemas/treatment.yaml:78]: Changed ticket to "Not Reported" per @tlicht3

File: gdcdictionary/schemas/treatment.yaml (hunk 3 above)
Comment (stilesj-uchicago) [reply gdcdictionary/schemas/treatment.yaml:206]: The ticket DICT-174 indicates 'Intratumoral'.

File: gdcdictionary/schemas/treatment.yaml (hunk 1 above)
Comment (wwysoc2) [reply gdcdictionary/schemas/treatment.yaml:214]: Fixed!

File: gdcdictionary/schemas/treatment.yaml [hunk 4]
```diff
@@ -100,12 +145,36 @@ properties:
         $ref:
           - "_terms_enum.yaml#/not_reported/common"
 
+  lesions_treated_number:
+    $ref:
+      - "_terms.yaml#/lesions_treated_number/common"
+    type: number
+
```
Comment (wteouchicago) [inline gdcdictionary/schemas/treatment.yaml:152]: can we add min of 0?

File: gdcdictionary/schemas/treatment.yaml [hunk 5]
```diff
@@ -100,12 +145,36 @@ properties:
...
 
+  number_of_fractions:
+    $ref:
+      - "_terms.yaml#/number_of_fractions/common"
+    type: number
+
+  prescribed_dose:
+    $ref:
+      - "_terms.yaml#/prescribed_dose/common"
+    type: number
```
Comment (wteouchicago) [inline gdcdictionary/schemas/treatment.yaml:167]: new prop, so maybe safe to add min of 0? (or whatever appropriate min).

File: gdcdictionary/schemas/treatment.yaml (hunk 1 above)
Comment (wteouchicago) [reply gdcdictionary/schemas/treatment.yaml:214]: looks good

File: gdcdictionary/schemas/treatment.yaml [hunk 6]
```diff
@@ -493,13 +570,39 @@ properties:
     type: integer
     minimum: 0
 
+  treatment_dose_max:
+    $ref:
+      - "_terms.yaml#/treatment_dose_max/common"
+    type: number
```
Comment (wteouchicago) [inline gdcdictionary/schemas/treatment.yaml:576]: new prop, so maybe safe to add min:0?

**Review by wteouchicago — APPROVED** [review]
noted a few areas where we can have additional min value checks. no blockers.

File: gdcdictionary/schemas/treatment.yaml (hunk 4 above)
Comment (wwysoc2) [reply gdcdictionary/schemas/treatment.yaml:153]: Great idea, added.

File: gdcdictionary/schemas/treatment.yaml (hunk 5 above)
Comment (wwysoc2) [reply gdcdictionary/schemas/treatment.yaml:169]: Added here too.

File: gdcdictionary/schemas/treatment.yaml (hunk 3 above)
Comment (wteouchicago) [reply gdcdictionary/schemas/treatment.yaml]: fixed in ticket and here.

File: gdcdictionary/schemas/treatment.yaml (hunk 6 above)
Comment (wwysoc2) [reply gdcdictionary/schemas/treatment.yaml:579]: Added here too.

File: gdcdictionary/schemas/treatment.yaml (hunk 4 above)
Comment (wteouchicago) [reply gdcdictionary/schemas/treatment.yaml:153]: looks good

File: gdcdictionary/schemas/treatment.yaml (hunk 5 above)
Comment (wteouchicago) [reply gdcdictionary/schemas/treatment.yaml:169]: looks good

File: gdcdictionary/schemas/treatment.yaml (hunk 6 above)
Comment (wteouchicago) [reply gdcdictionary/schemas/treatment.yaml:579]: looks good

**Review by stilesj-uchicago — APPROVED** [review]
LGTM. I think the description in the JIRA dict DICT-174 for treatment_arm description needs to be updated to say 'protocol_identifier'

**Comment by wwysoc2** [comment]
Looks like the "protocol_identifier" field has been figured out.  So I'll merge this.

**Pull request merged**